monitoring/remoteRun.ts

```diff
@@ -32,18 +44,29 @@ async function main() {
 		'ca-central-1',
 	];
 
-	const invokeSettledResults = await Promise.allSettled(
-		regionsToCheck.map((region) =>
-			invokeInRegion(region, 'cmp-monitoring-CODE'),
-		),
-	);
-
-	invokeSettledResults.map((result) => {
-		if (result.status == 'fulfilled') {
-			processResult(result.value);
-		} else {
-			console.log('Failed to get response: ', result.reason);
-		}
+	await prompt([
+		{
+			type: 'list',
+			name: 'stage',
+			message: 'Which environment would you like to test?',
+			choices: ['prod', 'code'],
+		},
+	]).then(async (userInput: RemoteRunCLIUserInput) => {
+		const invokeSettledResults = await Promise.allSettled(
+			regionsToCheck.map((region) =>
+				invokeInRegion(region, 'cmp-monitoring-CODE', userInput.stage),
+			),
+		);
+
+		invokeSettledResults.map((result) => {
```

Comment:
Could we aggregate the overall results and display that result in a nice way on the command line (also same comment about exit status here).